docs/release-notes/self-hosted/self-hosted-v10.0.0.md

```diff
@@ -0,0 +1,119 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v10.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools-legacy/releases/tag/6.3.0
+codacy_tools_version_new: https://github.com/codacy/codacy-tools-legacy/releases/tag/sh-1.1.0
+---
+
+# Self-hosted v10.0.0
+
+These release notes are for [Codacy Self-hosted v10.0.0](https://github.com/codacy/chart/releases/tag/10.0.0){: target="_blank"}, released on November 24, 2022.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v10.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v10.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the following versions:
+
+    -   [Codacy Analysis CLI 7.6.6](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.6.6)
+    -   [Codacy Coverage Reporter 13.10.15](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.10.15)
```

Comment:
Validate that these are the correct versions (this was copied from the [Codacy Self-hosted 9.0.0 release notes](https://docs.codacy.com/release-notes/self-hosted/self-hosted-v9.0.0/#upgrading-codacy-self-hosted)).

Reply:
We have created selfhosted tags ([self-hosted-10.0.0](https://github.com/codacy/codacy-analysis-cli/releases/tag/self-hosted-10.0.0)) which match the latest versions. Can we go with those? If we want the specific cli tool versions they are:
[Codacy Analysis CLI 7.7.4](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.7.4)
[Codacy Coverage Reporter 13.10.15](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.11.17)

Reply:
If we want to revert to the tag links:

```suggestion
1.  Update your Codacy command-line tools to the  versions with the Git tag `self-hosted-10.0.0`:

    -   [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli/releases/tag/self-hosted-10.0.0)
    -   [Codacy Coverage Reporter](https://github.com/codacy/codacy-coverage-reporter/releases/tag/self-hosted-10.0.0)
```

Reply:
If we want to stay with the [same versioning syntax we use in SH 9](https://docs.codacy.com/release-notes/self-hosted/self-hosted-v9.0.0/#upgrading-codacy-self-hosted):

```suggestion
1.  Update your Codacy command-line tools to the following versions:

    -   [Codacy Analysis CLI 7.7.4](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.7.4)
    -   [Codacy Coverage Reporter 13.10.15](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.10.15)
```